src/components/SideMenuItem/SideMenuItem.jsx

```diff
@@ -0,0 +1,24 @@
+import React, { PureComponent } from 'react';
+
+import SideMenuItemWrapper from './SideMenuItemWrapper';
+
+const sideMenuItemClick = (id, sportItemClick) => () => {
```

Comment:
Is there a reason for this separate function?

Reply:
The only reason for this is to call handler with provided id, not the event object

Reply:
✔️